numpy/meson.build

```diff
@@ -223,17 +223,19 @@ null_dep = dependency('', required : false)
 atomic_dep = null_dep
 code_non_lockfree = '''
   #include <stdint.h>
+  #include <stddef.h>
+  #include <stdatomic.h>
   int main() {
    struct {
      void *p;
      uint8_t u8v;
    } x;
    x.p = NULL;
    x.u8v = 0;
-   uint8_t res = __atomic_load_n(x.u8v, __ATOMIC_SEQ_CST);
-   __atomic_store_n(x.u8v, 1, ATOMIC_SEQ_CST);
-   void *p = __atomic_load_n(x.p, __ATOMIC_SEQ_CST);
-   __atomic_store_n((void **)x.p, NULL, __ATOMIC_SEQ_CST)
+   uint8_t res = __atomic_load_n(&x.u8v, __ATOMIC_SEQ_CST);
+   __atomic_store_n(&x.u8v, 1, __ATOMIC_SEQ_CST);
+   void *p = __atomic_load_n((void **)x.p, __ATOMIC_SEQ_CST);
+   __atomic_store_n((void **)x.p, NULL, __ATOMIC_SEQ_CST);
```

Comment:
Huh, I guess this never worked. I believe this code was copied from SciPy as the comment above alludes, can you also check it's correct there?

Reply:
Checking over on SciPy, it looks like it was correct over there and this just never worked.